src/main/webapp/definitions/alfresco-search-sql.yaml

```diff
@@ -0,0 +1,316 @@
+swagger: '2.0'
+info:
+  description: |
+    **Search SQL API**
+    A Solr SQL passthrough, this feature provides the ability to use SQL to query Solr.
+  version: '1'
+  title: Alfresco Insight Engine REST API
+basePath: /alfresco/api/-default-/public/search/versions/1/sql
```

Comment:
Having the SQL endpoint publicly available is not ideal (as it won't work for community) but we discussed that already.

Reply:
Just saw the note below. 👍